src/roman_datamodels/datamodels.py

```diff
@@ -242,10 +242,17 @@ def get_crds_parameters(self):
         -------
         dict
         """
-        return {
+        crds_header = {
             key: val for key, val in self.to_flat_dict(include_arrays=False).items()
```

Comment:
Does `to_flat_dict` return keys that start with "roman."?  If not then we'll need to modify the key here.

Reply:
I thought that was what you were requesting due to other parts of CRDS looking at the raw asdf, where roman is the top attribute. I can remove roman, but is that what you want? All the current selection criteria include it.

Reply:
Right, that's what I was confirming, CRDS does want the keys to start with `roman.`.  I wasn't sure if `to_flat_dict` would do that since in `DataModel.__init__` we're pulling out the roman node and assigning it to `_instance`.  But I see now that the `to_flat_dict` implementation is adding "roman." back in:

```python
        if include_arrays:
            return dict(('roman.' + key, convert_val(val)) for (key, val) in self.items())
        else:
            return dict(('roman.' + key, convert_val(val)) for (key, val) in self.items()
                        if not isinstance(val, np.ndarray))
                        ```